### PR TITLE
tiflash: Refine the config doc of tiflash

### PR DESCRIPTION
**Pull request by JaySon-Huang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

* Add more details about tiflash && tiflash-proxy logging config.
* Remove `raftstore.store-batch-retry-recv-timeout` from tiflash-proxy config. This item have been removed from tiflash-proxy since v6.5

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Lloyd-Pottiger — APPROVED** [review]

**Review by CalvinNeo — APPROVED** [review]
lgtm

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -155,23 +155,27 @@ delta_index_cache_size = 0
...
     data-dir = "/tidb-data/tiflash-9000/flash"
     ## proxy 配置文件路径
     config = "/tidb-deploy/tiflash-9000/conf/tiflash-learner.toml"
     ## proxy log 路径
     log-file = "/tidb-deploy/tiflash-9000/log/tiflash_tikv.log"
-    ## proxy 的 log 级别 (支持 "trace"、"debug"、"info"、"warn"、"error"). 默认是 "info"
-    # log-level = "info" 
 
 [logger]
+    ## 注意，以下参数只对 tiflash.log，tiflash_error.log 生效。TiFlash proxy 的日志参数配置需要在 tiflash-learner.toml 中指定。
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:168]:
```suggestion
    ## 注意，以下参数只对 tiflash.log、tiflash_error.log 生效。TiFlash proxy 的日志参数配置需要在 tiflash-learner.toml 中指定。
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -155,23 +155,27 @@ delta_index_cache_size = 0
...
+
     ## log 级别（支持 "trace"、"debug"、"info"、"warn"、"error"），默认是 "info"
     level = "info"
+    ## tiflash 日志
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:172]:
```suggestion
    ## TiFlash 日志
```

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -155,23 +155,27 @@ delta_index_cache_size = 0
...
     log = "/tidb-deploy/tiflash-9000/log/tiflash.log"
+    ## tiflash 错误日志。对于 "warn", "error" 级别的日志，会额外输出到该日志文件中。
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:174]:
```suggestion
    ## TiFlash 错误日志。对于 "warn"、"error" 级别的日志，会额外输出到该日志文件中。
```

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -155,23 +155,27 @@ delta_index_cache_size = 0
...
     errorlog = "/tidb-deploy/tiflash-9000/log/tiflash_error.log"
     ## 单个日志文件的大小，默认是 "100M"
     size = "100M"
-    ## 最多保留日志文件个数，默认是 10
+    ## 最多保留日志文件个数，默认是 10。对于 tiflash 日志和 tiflash 错误日志各自最多保留 `count` 个。
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:178]:
```suggestion
    ## 最多保留日志文件个数，默认是 10。对于 TiFlash 日志和 TiFlash 错误日志各自最多保留 `count` 个日志文件。
```

File: tiflash/tiflash-configuration.md [hunk 5]
```diff
@@ -257,19 +261,33 @@ delta_index_cache_size = 0
 
 ### 配置文件 tiflash-learner.toml
 
+tiflash-learner.toml 中的功能参数和 TiKV 基本一致，可以参照[TiKV 配置](/tikv-configuration-file.md)来进行配置，下面只列出常用的部分参数。需要注意的是：
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:264]:
```suggestion
`tiflash-learner.toml` 中的功能参数和 TiKV 基本一致，可以参照 [TiKV 配置](/tikv-configuration-file.md)来进行配置。下面只列了常用的部分参数。需要注意的是：
```

File: tiflash/tiflash-configuration.md [hunk 6]
```diff
@@ -257,19 +261,33 @@ delta_index_cache_size = 0
...
+`tiflash-learner.toml` 中的功能参数和 TiKV 基本一致，可以参照 [TiKV 配置](/tikv-configuration-file.md)来进行配置。下面只列了常用的部分参数。需要注意的是：
+
+1. 相对于 TiKV，TiFlash proxy 新增了 `raftstore.snap-handle-pool-size` 参数。
+2. `key` 为 `engine` 的 `label` 是保留项，不可手动配置。
+
 ```toml
-[server]
-    engine-addr = 外部访问 TiFlash coprocessor 服务的地址
+[log]
+    ## 从 v5.4.0 版本开始引入，proxy 的 log 级别 (支持 "trace"、"debug"、"info"、"warn"、"error"). 默认是 "info"
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:271]:
```suggestion
    ## Proxy 的 log 级别，可选值为 "trace"、"debug"、"info"、"warn"、"error"，默认值为 "info"。从 v5.4.0 版本开始引入。
```

File: tiflash/tiflash-configuration.md [hunk 7]
```diff
@@ -257,19 +261,33 @@ delta_index_cache_size = 0
...
+    # level = "info"
+[log.file]
+    ## 可保留的 log 文件的最大数量。从 v5.4.0 版本开始引入
+    ## 如果未设置本参数或把此参数设置为默认值 `0`，TiKV 会保存所有的日志文件；
+    ## 如果把此参数设置为非 `0` 的值，TiKV 最多会保留 `max-backups` 中指定的数量的旧日志文件。比如，如果该值设置为 `7`，TiKV 最多会保留 7 个旧的日志文件。
+    # max-backups = 0
+    ## 保留 log 文件的最长天数。从 v5.4.0 版本开始引入
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:278]:
```suggestion
    ## 可保留的 log 文件的最大数量。从 v5.4.0 版本开始引入。
    ## 如果未设置该参数或把该参数设置为默认值 `0`，TiKV 会保存所有的日志文件；
    ## 如果把此参数设置为非 `0` 的值，TiKV 最多会保留 `max-backups` 中指定数量的旧日志文件。比如，如果该值设置为 `7`，TiKV 最多会保留 7 个旧的日志文件。
    # max-backups = 0
    ## 保留 log 文件的最长天数。从 v5.4.0 版本开始引入。
```

File: tiflash/tiflash-configuration.md [hunk 8]
```diff
@@ -257,19 +261,33 @@ delta_index_cache_size = 0
...
+    # level = "info"
+[log.file]
+    ## 可保留的 log 文件的最大数量。从 v5.4.0 版本开始引入。
+    ## 如果未设置该参数或把该参数设置为默认值 `0`，TiKV 会保存所有的日志文件；
+    ## 如果把此参数设置为非 `0` 的值，TiKV 最多会保留 `max-backups` 中指定数量的旧日志文件。比如，如果该值设置为 `7`，TiKV 最多会保留 7 个旧的日志文件。
+    # max-backups = 0
+    ## 保留 log 文件的最长天数。从 v5.4.0 版本开始引入。
+    ## 如果未设置本参数或把此参数设置为默认值 `0`，TiKV 会保存所有的日志文件。
+    ## 如果把此参数设置为非 `0` 的值，在 `max-days` 之后，TiKV 会清理过期的日志文件。
+    # max-days = 0
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:281]: 这里是特意注释掉的吗？
```suggestion
    level = "info"
[log.file]
    ## 可保留的 log 文件的最大数量。从 v5.4.0 版本开始引入。
    ## 如果未设置该参数或把该参数设置为默认值 `0`，TiKV 会保存所有的日志文件；
    ## 如果把此参数设置为非 `0` 的值，TiKV 最多会保留 `max-backups` 中指定数量的旧日志文件。比如，如果该值设置为 `7`，TiKV 最多会保留 7 个旧的日志文件。
    max-backups = 0
    ## 保留 log 文件的最长天数。从 v5.4.0 版本开始引入。
    ## 如果未设置本参数或把此参数设置为默认值 `0`，TiKV 会保存所有的日志文件。
    ## 如果把此参数设置为非 `0` 的值，在 `max-days` 之后，TiKV 会清理过期的日志文件。
    max-days = 0
```

Comment (JaySon-Huang) [reply tiflash/tiflash-configuration.md:281]: LGTM

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**